### DISCUSSION TITLE
Set default hover setting for Timeseries panel from "Single" to "All"

### DISCUSSION THREAD

**Discussion by nikobearrr** [grafana/grafana#46992] [Dashboards, Panels & Visualizations]
In the old Graph panel `All` used to be the default.
Now, in the new TimeSeries panel `Single` is default.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
